<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Behavior of function</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_10">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabs_of_functions">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Plane(cartesian)</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_of_cartesian_le">
             <property name="text">
              <string>y=</string>
             </property>
             <property name="buddy">
              <cstring>cartesian_le</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="cartesian_le">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Plane(polar)</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_of_polar_le">
             <property name="text">
              <string>r=</string>
             </property>
             <property name="buddy">
              <cstring>polar_le</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="polar_le"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>angle(variable)</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_of_FROM_of_polar">
                <property name="text">
                 <string>from</string>
                </property>
                <property name="buddy">
                 <cstring>spinb_FROM_of_polar</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="spinb_FROM_of_polar">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="minimum">
                 <double>-1000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_of_TO_of_polar">
                <property name="text">
                 <string>to</string>
                </property>
                <property name="buddy">
                 <cstring>spinb_TO_of_polar</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="spinb_TO_of_polar">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="minimum">
                 <double>-1000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Parametered</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_of_y_parametered_le">
               <property name="text">
                <string>y=</string>
               </property>
               <property name="buddy">
                <cstring>y_parametered_le</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="y_parametered_le"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_of_x_parametered_le">
               <property name="text">
                <string>x=</string>
               </property>
               <property name="buddy">
                <cstring>x_parametered_le</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="x_parametered_le"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>t(parameter)</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_of_FROM_of_parametered">
                <property name="text">
                 <string>from</string>
                </property>
                <property name="buddy">
                 <cstring>spinb_FROM_of_parametered</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="spinb_FROM_of_parametered">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="minimum">
                 <double>-1000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_of_TO_of_parametered">
                <property name="text">
                 <string>to</string>
                </property>
                <property name="buddy">
                 <cstring>spinb_TO_of_parametered</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="spinb_TO_of_parametered">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="minimum">
                 <double>-1000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
           <zorder></zorder>
           <zorder></zorder>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBoxOfFunction_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>134</width>
         <height>92</height>
        </size>
       </property>
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Graphics&lt;br&gt; size</string>
            </property>
            <property name="buddy">
             <cstring>slider_graphic_size</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_graphic_size">
            <property name="minimum">
             <number>151</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="singleStep">
             <number>2</number>
            </property>
            <property name="value">
             <number>175</number>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Scale</string>
            </property>
            <property name="buddy">
             <cstring>sliderRatio</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderRatio">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>40</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="pageStep">
             <number>2</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_show_grid">
          <property name="text">
           <string>Show grid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbCompute">
          <property name="text">
           <string>Compute</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabW_of_graphics">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Function</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>678</width>
             <height>1026</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="function_graphic_label">
              <property name="text">
               <string>Label_of_function_graphic</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Derivation</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="derivation_graphic_label">
          <property name="text">
           <string>Label_of_derivation_graphic</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Function and Derivation</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="function_and_derivation_graphic_label">
          <property name="text">
           <string>Label_of_function_and_derivation_graphic</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pbCompute</tabstop>
  <tabstop>sliderRatio</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
